src/publish_control_board_rev3.cpp

```diff
@@ -67,6 +67,8 @@ void PublishControlBoardRev3::publish_steering_message(const sensor_msgs::Joy::C
   float range_scale;
   if (vehicle_type == VEHICLE_4 || vehicle_type == VEHICLE_6 || vehicle_type == LEXUS_RX_450H)
     range_scale = 1.0;
+  else if (vehicle_type == FREIGHTLINER_CASCADIA)
+    range_scale = 1.0;  // testing
```

Comment:
This can be combined with the `if` above this. Also, please remove comment.

Reply:
I wanted to test it on the truck before combining it in the if.